docs/topics/test-system.md

```diff
@@ -6,16 +6,31 @@ You can find all code for running test test systems in the `/opt/test-systems` d
 This directory is owned by the `mediawiki` user.
 Each test system is a copy of the `example` docker-compose setup, with customized env vars and settings.
 
-Two test systems are maintained, one for the current "latest" major release, and one for the "previous" major release.
+Four optional test systems are maintained.
+These may or may not be running at any given time, as they are only intended for use during product verification during the release process.
+Engineers can start and stop these
+
+**Default Wikibase**
 
-- Latest release (internal ports 83**)
-  - https://wikibase-product-testing.wmcloud.org (8380 internal)
-  - https://wikibase-query-testing.wmcloud.org (8381 internal)
-  - https://wikibase-qs-testing.wmcloud.org (8382 internal)
 - Previous release (internal ports 82**)
   - https://wikibase-product-testing-previous.wmcloud.org (8280 internal)
   - https://wikibase-query-testing-previous.wmcloud.org (8281 internal)
   - https://wikibase-qs-testing-previous.wmcloud.org (8282 internal)
+- Latest release (internal ports 83**)
+  - https://wikibase-product-testing.wmcloud.org (8380 internal)
+  - https://wikibase-query-testing.wmcloud.org (8381 internal)
+  - https://wikibase-qs-testing.wmcloud.org (8382 internal)
+
+**Federated Properties**
+
+- Previous release with fed props (internal ports 84**)
+  - https://wikibase-product-testing-fedprops-previous.wmcloud.org (8480 internal)
+  - https://wikibase-query-testing-fedprops-previous.wmcloud.org (8481 internal)
+  - https://wikibase-qs-testing-fedprops-previous.wmcloud.org (8482 internal)
+- Latest release with fed props (internal ports 85**)
+  - https://wikibase-product-testing-fedprops.wmcloud.org (8580 internal)
```

Comment:
These seem down which is fine, not sure if it would make sense for anyone merging this to use the script to set it up too rather than the author? (I haven't tried the script out)

Reply:
Heh, these were up yesterday *looks at them*

Reply:
Oh wait sorry, yes, I didnt bother setting up this particular test system, as it was not needed for the release that I am currently working through.
If you want to give it a go to setup this instance go for it, but we may run into memory issues and you might need to switch off one of the other systems! (Which I think is fine)